### DISCUSSION TITLE
Remix previous version installation error

### DISCUSSION THREAD

**Discussion by 9mean2** [remix-run/remix#7740] [Q&A]
There was no error when installing the latest version of Remix, but if you proceed with npx create-remix@1.15.0 as shown below, the installation fails with the error below.

"Creating your app…🚨 The provided template must be a Remix project with a package.json file, but that file does not exist in /home/goomin/desktop/test/app/package.json."

What should I do and what is the problem? 😖

I tried it on Mac and Ubuntu, but it still shows the same error.

```
npx create-remix@1.15.0
? Where would you like to create your app? ./
? What type of app do you want to create? Just the basics
? Where do you want to deploy? Choose Remix App Server if you're unsure; it's 
easy to change deployment targets. Vercel
? TypeScript or JavaScript? TypeScript
? Do you want me to run `npm install`? Yes
⠧ Creating your app…🚨 The provided template must be a Remix project with a `package.json` file, but that file does not exist in /home/goomin/바탕화면/test/package.json.
```

**Comment by Erij-Maherzia-BEN-BRAHIM** [comment]
It's written that the `package.json` file is missing.
You can create a basic package.json file using the following command:
```
npm init -y
```
This will generate a default package.json file with default values.
